### PR TITLE
fs-extra: Add EOL to WriteOptions.

### PR DESCRIPTION
**Pull request by aomarks**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jprichardson/node-fs-extra/blob/master/docs/writeJson.md (added in 4.0.2: https://github.com/jprichardson/node-fs-extra/blob/master/CHANGELOG.md#402--2017-09-12)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by aomarks** [comment]
cc @alan-agius4 @midknight41 @shiftkey @mees- 

(dt-bot doesn't seem to be working)

**Review by shiftkey — APPROVED** [review]

**Review by alan-agius4 — APPROVED** [review]

**Review by stramel — APPROVED** [review]

**Comment by andy-ms** [comment]
Approved by a listed owner. PR ready to merge pending express review by a maintainer.

**Pull request merged**
